wallet/wallet-cli-commands/src/errors.rs

```diff
@@ -20,7 +20,7 @@ use wallet_controller::types::GenericCurrencyTransferToTxOutputConversionError;
 use wallet_rpc_client::{handles_client::WalletRpcHandlesClientError, rpc_client::WalletRpcError};
 use wallet_rpc_lib::RpcError;
 
-#[derive(thiserror::Error, Debug)]
+#[derive(thiserror::Error, derive_more::Debug)]
```

Comment:
FYI: the standard `Debug` macro requires `N` to implement `Debug` as well, so basically it never works. `derive_more::Debug` is more clever and puts no requirements on `N`.